components/layouts/Conversation.tsx

```diff
@@ -173,12 +173,17 @@ export default function Conversation({ jeuneChat, onBack }: ConversationProps) {
       <form
         data-testid='newMessageForm'
         onSubmit={sendNouveauMessage}
-        className='w-full bg-blanc p-3 flex'
+        className='w-full bg-blanc p-3 flex items-end'
       >
-        <input
-          type='text'
+        <label htmlFor='input-new-message' className='sr-only'>
+          Nouveau message
```

Comment:
bien vu le label! 
 pour distinguer d'un nouveau message à lire
```suggestion
message à envoyer
```